pytradfri/smart_task.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, gateway: Gateway, raw: TypeRaw) -> None:
         """Initialize the class."""
         super().__init__(raw)
         self._gateway = gateway
-        self._delta_time_gateway_local: timedelta = timedelta(0)
+        self.delta_time_gateway_local: timedelta = timedelta(0)
```

Comment:
Do we need the type annotation?

Reply:
We did not, at least mypy didn't react to me removing it.